### ISSUE TITLE
About loss in training

### ISSUE THREAD

**Issue by Shualite** [motokimura/PyTorch_Gaussian_YOLOv3#43]
![image](https://user-images.githubusercontent.com/50547280/92206646-6d1a7680-eeba-11ea-9fe6-66511d546e6b.png)
![image](https://user-images.githubusercontent.com/50547280/92206718-8c190880-eeba-11ea-833e-338ce5772789.png)

The training process above is mine,

1) I'm so confused about the loss, which is too big. And I find that the author use clip_grad_norm. It seems to control the too big loss. 
2) During training, learning rate is 0.00000. Is that ok?

**Comment by motokimura** [comment]
Sorry for my late reply.

1. `conf` loss term is high in the early phase of training but it' normal.
2. It's fine. Usually warm-up lr scheduling is used for YOLO models. Without warm-up scheduling, the loss diverges.
